### PR TITLE
Single file build script

### LINKED ISSUE #29: Standalone build?
Hello!
I would really love to use react-leaflet in my project but i don't have a require.js there and i can't have it installed for various reasons. I just can't. 

I can't see any standalone build of react-leaflet in the repo but i've seen a lot or react (and js in generall) libs made in two version - for AMD and standalone so i assume that's doable. I'm not really good in JS and i don't know require.js so i'm totally not sure if that's possible but - is that possible for you to add a standalone version ? Or can i somehow build such thing myself?

Thanks!

### PR DESCRIPTION
**Pull request by gambit3**
This is a custom build script I use to build a single (but unminified) version of react-leaflet. It should fix #29

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,7 +13,8 @@
     "dev": "npm run watch & npm run onchange",
     "prepublish": "npm ls && npm test",
     "example:compile": "gulp",
-    "example:watch": "gulp watch:example"
+    "example:watch": "gulp watch:example",
+    "build_single": "node single_build_globals.js"
```
Comment (luqmaan) [inline package.json]: For consistency should this be `build:single`?

File: single_build_globals.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+var browserify = require('browserify');
+var babelify = require("babelify");
```
Comment (luqmaan) [inline single_build_globals.js]: Use `'` vs `"` consistently.

File: package.json (hunk 1 above)
Comment (gambit3) [inline package.json]: Sure. The script should probably also live somewhere else and always build the same file. I didn't know how to organise it to be honest, just wanted to share the build script.

**Comment by PaulLeCam** [comment]
Thanks but I plan to move over to Webpack for the build, I'll make it export to UMD.

**Pull request closed without merging**